3D Chess/Assets/Scripts/GameState.cs

```diff
@@ -95,4 +97,20 @@ private PieceEntity FindPieceEntity(Vector3 position)
                          position.x, position.y, position.z)
                  .FirstOrDefault();
     }
+
+    private async Task<Realm> CreateRealmAsync()
+    {
+        var app = App.Create("3d_chess-sjdkk");
+
+        // For this example we can just create a new random user to play the game since all users access the same game.
+        // They have to be distinct though, so anonymous credentials are not an option.
+        var email = Guid.NewGuid().ToString();
+        var password = "password";
+        await app.EmailPasswordAuth.RegisterUserAsync(email, password);
+
+        var user = await app.LogInAsync(Credentials.EmailPassword(email, password));
+        var syncConfiguration = new SyncConfiguration("3d_chess_partition_key", user);
```

Comment:
It might be a good idea to extend the example to show a welcome screen where the user enters the game id, which can be used as a partition key. That way we can support multiple games in parallel.

Reply:
Et voila: bf9d83f.

I thought about getting even more sophisticated and adding a back button or showing a list of available games.
But then I'd need one partition per game and probably one (or a separate Realm) for the list of games and I figured that's a nice tutorial in itself about multiple Realms, using ObjectSchemas and partitions. 👍